objects/obj_shop/Mouse_50.gml

```diff
@@ -25,18 +25,17 @@ if (mouse_y>=yy+76) and (mouse_y<yy+104) and (obj_controller.cooldown<=0){
     instance_create(50,50,obj_shop);
 }
 
+draw_rectangle(xx+957,yy+76,xx+1062,yy+104,0);
+draw_rectangle(xx+1068,yy+76,xx+1150,yy+104,0);
```

Comment:
tried to fix this:

![image](https://github.com/OH296/ChapterMaster/assets/63882831/36a5899e-27d1-40f8-b779-f0cd1613a54a)

increased the second xx from 1136 to 1150, but the buggy textbox didn't seem to change at all

Reply:
@GalacticChimp this is the wrong draw area correct edit but should have been in the file scr_ui_advisors. Some duplication going on i'll take a look at doing a refactor at some point